keyboards/geminate60/geminate60.c

```diff
@@ -0,0 +1,16 @@
+  /* Copyright %YEAR% %YOUR_NAME% 

```

Comment:
Placeholder tokens need filling out

Reply:
Oh, I see.
I've modified all the licenses.